src/core/api/public_api.ts

```diff
@@ -667,7 +669,9 @@ class Player extends EventEmitter<IPublicAPIEvent> {
 
       const segmentRequestOptions = { lowLatencyMode,
                                       maxRetry: requestConfig.segment?.maxRetry,
-                                      requestTimeout: requestConfig.segment?.timeout };
+                                      requestTimeout: requestConfig.segment?.timeout,
+                                      requestConnectionTimeout:
```

Comment:
Why renaming it here but not in the `manifestRequestSettings`?

Reply:
Didn't finish yet the segment part